### ISSUE TITLE
Request to add external GPS receivers to the docs

### ISSUE THREAD

**Issue by PeterPetrik** [MerginMaps/docs#124]
If you have external GPS that is not listed in the official documentation, please let us know as comment to this issue and we will add it... (OR create Pull Request and add it yourself :))

**Comment by gilles-morain** [comment]
Hi, I was able to connect Input 1.3.0 (Android) to a an Emlid Reach M+ (not a surprise since Reach RS+ is listed) using several methods:
- direct Bluetooth connection
- Android mock location > Bluetooth > [GPS Connector app](https://play.google.com/store/apps/details?id=de.pilablu.gpsconnector)
- Android mock location > Wifi > [NTRIP Lefebure Client](https://play.google.com/store/apps/details?id=com.lefebure.ntripclient)
No luck yet with iOS... However, since NMEA over TCP seems to work, I would be happy to test an option within Input for a direct connection over TCP !

**Comment by PeterPetrik** [comment]
For reference, connection over TCP is https://github.com/lutraconsulting/input/issues/1893

**Comment by jdelbridge** [comment]
I’ve been using the Garmin GLO 2 on an iPad with the Input app without trouble for a couple months.

**Comment by LiliiaPleskachBRRC** [comment]
Hi, we are using the [UNI-GR1 marXact GPS receivers](https://marxact.com/) on Android tablets with the Input app, works perfect!

**Comment by PeterPetrik** [comment]
Tested with Emlid Reach RX https://www.linkedin.com/feed/update/urn:li:activity:6993131837707452416/ on android

**Comment by Jharbers1980** [comment]
We use Carlson GNSS receivers (Brx7 & XML2) to take measurements in Mergin Maps, which works well on Android! In combination with Carlson Layout, this application can overrides the mock location of the tablet's internal GPS.

**Comment by jolivart** [comment]
Could you please add these devices from ArduSimple?

- https://www.ardusimple.com/product/rtk-calibrated-surveyor-kit/
- https://www.ardusimple.com/product/rtk-handheld-surveyor-kit/
- https://www.ardusimple.com/product/simplertk2blite-bt-case-kit/

Thanks!

**Comment by jolivart** [comment]
All the devices can work directly with Mergin Maps.
They can be used in parallel with GNSS Master app for Android to get RTK corrections.
We have prepared a tutorial:
https://www.ardusimple.com/how-to-connect-ardusimple-kit-to-mergin-maps-for-centimeter-accuracte-mapping/

Thanks!

**Comment by gef431** [comment]
Can you trimble r4s. Receiver?

**Comment by gef431** [comment]
Can you add trimble r4s receiver?

**Comment by alex-cit** [comment]
Hi @gef431 , could you please specify some details about the usage? Does it work through Trimble Mobile Manager app (like R1 and R2) or something else? Do you use it on Android or iOS? Thanks!

**Comment by gef431** [comment]
Yes. It work with TMM.Can I configure the application to use TMM position?
B rgds

Yahoo Mail : Recherchez, organisez, maîtrisez 
 
  Le dim., nov. 3, 2024 à 10:09 PM, Alexandra Bucha ***@***.***> a écrit:   


Hi @gef431 , could you please specify some details about the usage? Does it work through Trimble Mobile Manager app (like R1 and R2) or something else? Do you use it on Android or iOS? Thanks!

**Comment by gef431** [comment]
Yahoo Mail : Recherchez, organisez, maîtrisez 
 
  Le dim., nov. 3, 2024 à 10:09 PM, Alexandra Bucha ***@***.***> a écrit:   


Hi @gef431 , could you please specify some details about the usage? Does it work through Trimble Mobile Manager app (like R1 and R2) or something else? Do you use it on Android or iOS? Thanks!

**Comment by gef431** [comment]
Ho
Use with android. But have not access to select  TMM !
B rgds

Yahoo Mail : Recherchez, organisez, maîtrisez 
 
  Le dim., nov. 3, 2024 à 10:09 PM, Alexandra Bucha ***@***.***> a écrit:   


Hi @gef431 , could you please specify some details about the usage? Does it work through Trimble Mobile Manager app (like R1 and R2) or something else? Do you use it on Android or iOS? Thanks!

**Comment by HydretudesOI** [comment]
I have a Spectra SP100 (Trimble) GNSS receiver. I'm using Trimble Mobile Manager (TMM) to connect to the receiver and to TERIA as the NTRIP correction service. I also enabled Developer Mode in TMM so I could activate the Android Mock Location feature through Trimble Mobile Manager.

My goal is to use the SP100 with Mergin Maps.

However, when I select "Internal receiver" in Mergin Maps, the positioning accuracy is nowhere near centimeter-level. Instead, it ranges between 20 and 80 meters, as if the app is using the phone's internal GPS rather than the SP100.

If I select "Internal receiver (fusion)", I don't get any location data at all—the app doesn't display any position.

Could you help me understand what might be causing this issue and how to configure Mergin Maps correctly so it uses the SP100 with the RTK corrections provided by TMM? Even with GNSS Master it doesn't work. Dans directly by bluetooh, I don't even have the coordinates

**Comment by tomasMizera** [comment]
Hi @HydretudesOI, I haven't used TMM as a mock position provider myself, but we're adding native TMM support to the mobile app. The upcoming release introduces a new position provider connection method for direct communication with Trimble devices, see https://github.com/MerginMaps/mobile/pull/4641.

Once it's out, I'd recommend switching to this method instead of the mock workaround.
